data/json/effects.json

```diff
@@ -494,6 +494,31 @@
     "show_in_info": true,
     "blood_analysis_description": "Poison"
   },
+  {
+    "type": "effect_type",
+    "id": "tpollen",
+    "name": [ "Hard of Breath" ],
+    "desc": [ "The pollen makes it hard to breath!" ],
+    "miss_messages": [ [ "You cough out pollen.", 1 ] ],
+    "rating": "bad",
+    "harmful_cough": true,
+    "pain_sizing": true,
+    "hurt_sizing": true,
+    "main_parts_only": true,
+    "base_mods": {
+      "per_mod": [ -2, -1 ],
+      "dex_mod": [ -1, -1 ],
+      "str_mod": [ -2, 0 ],
+      "pain_min": [ 2 ],
+      "pain_chance": [ 150, 900 ],
+      "hurt_min": [ 2 ],
+      "hurt_chance": [ 450, 2700 ],
+      "speed_mod": [ -10 ],
+      "cough_chance": [ 10 ]
+    },
+    "show_in_info": true,
+    "blood_analysis_description": "Triffid Poison"
```

Comment:
The blood analysis machine doesn't know what triffids are.

Reply:
Ah; completely fair, should it just be... poison?

Reply:
Maybe something like "unknown organic poison"?

Reply:
Changed to poison